man/man8/zfs.8

```diff
@@ -441,6 +441,16 @@ For non-snapshots, the compression ratio achieved for the \fBused\fR space of th
 .sp
 .ne 2
 .na
+\fB\fBcreatetxg\fR
+.ad
+.sp .6
+.RS 4n
+The transaction group (TXG) in which the dataset or bookmark was created. It is suitable for ordering a list of snapshots, e.g. for incremental \fBsend\fR & \fBrecv\fR.
```

Comment:
Adressed in https://github.com/zfsonlinux/zfs/pull/6102/commits/a4da85a905da363c8adcd6225bea69ae3a949d10

Should I squash each time I respond to a comment or rather squash right before merge?